### DISCUSSION TITLE
The conversion could not be started, or failed due to an error.

### DISCUSSION THREAD

**Discussion by fedex03** [firefly-iii/firefly-iii#8459] [Firefly III] [answered]
### Support guidelines

- [X] I've read the [support guidelines](https://github.com/firefly-iii/firefly-iii/blob/main/.github/support.md)

### I've found a bug and checked that ...

- [ ] ... [the documentation](https://docs.firefly-iii.org/) does not mention anything about my problem
- [ ] ... there are no open or closed issues that are related to my problem
- [ ] ... it's [definitely a Firefly III issue, not me](https://github.com/firefly-iii/firefly-iii/blob/main/.github/its_you_not_me.md)

### Description

Hi there,

I always imported expenses from Revolut using Firefly III Data Import Tool, (v1.4.2).
Today I tried to import new expenses but I get this error:

` The conversion could not be started, or failed due to an error. Please check the log files. Sorry about this :(
Error(s)

    Line #0:
        (2)Could not parse CSV: Could not decode JSON line #198: Malformed UTF-8 characters, possibly incorrectly encoded
        (2)No transactions found in CSV file.

`

I took a look into the CSV file at the line zero but I did notice something bad.

Could you please help with with some hint to find the error?

Thanks in advance.

### Debug information

Nothing

**Comment by JC5** [comment]
What command are you using to start the import? Are you using the UI?

**Comment by fedex03** [comment]
I'm using the UI, and the import setting I always used.

**Reply by JC5** [reply]
Well the error feels obvious. Could you try disabling the UTF8 conversion, if it is enabled?

**Reply by fedex03** [reply]
Hi,

the conversion is disabled.

**Reply by JC5** [reply, marked as answer]
Then there are probably invalid characters in the file. The check is pretty foolproof. You could try using Notepad++ to save the file as ascii.

**Reply by fedex03** [reply]
I found some invalid characters in the file, but they are not in the row 0.

Let me try with Notepad++

Thanks!

**Reply by JC5** [reply]
That makes sense, it will try the file in one go, it will always say they're in row zero.

**Reply by fedex03** [reply]
I found the invalid characters using this regulare expression into the Find section of notepad++: "[^\x00-\x7F]+"

There were some 'è' in the file.

Thank you for your help!
